testsuite/features/support/remote_node.rb

```diff
@@ -327,9 +328,9 @@ def client_public_ip
 
   # Extract the OS version and OS family
   # We get these data decoding the values in '/etc/os-release'
-  def get_os_version
-    os_family_raw, code = run('grep "^ID=" /etc/os-release', check_errors: false)
-    os_family_raw, code = run('sw_vers --productName', check_errors: false) if code.nonzero?
+  def get_os_version(real_host: false)
+    os_family_raw, code = run('grep "^ID=" /etc/os-release', run_in_container: !real_host, check_errors: false)
+    os_family_raw, code = run('sw_vers --productName', run_in_container: !real_host, check_errors: false) if code.nonzero?
```

Comment:
```suggestion
  def get_os_version(run_in_container: false)
    os_family_raw, code = run('grep "^ID=" /etc/os-release', run_in_container: run_in_container, check_errors: false)
    os_family_raw, code = run('sw_vers --productName', run_in_container: run_in_container, check_errors: false) if code.nonzero?
```

Reply:
Oscar also prefers if we keep same parameter name. I don't like bending the semantics, but okay, I will comply.